tests/integration/email_embed_b64_images_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package integration
+
+import (
+	"context"
+	"testing"
+
+	issues_model "code.gitea.io/gitea/models/issues"
+	repo_model "code.gitea.io/gitea/models/repo"
+	"code.gitea.io/gitea/models/unittest"
+	user_model "code.gitea.io/gitea/models/user"
+	"code.gitea.io/gitea/modules/setting"
+	mail "code.gitea.io/gitea/services/mailer"
+	"code.gitea.io/gitea/tests"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEmailEmbedBase64Images(t *testing.T) {
```

Comment:
If I understand correctly, this test could be a "unit test" (in `services/mailer/mail_xxx_test.go`), but not a "integration test"? Then no need to expose `mailCommentContext`?